TwitchPlaysAssembly/Src/UI/TwitchModule.cs

```diff
@@ -492,10 +492,13 @@ public ClaimResult TryClaim(string userNickName, bool viewRequested = false, boo
 
 		// Would violate the claim limit ⇒ queue
 		if (TwitchGame.Instance.Modules.Count(md => md.PlayerName != null && md.PlayerName.EqualsIgnoreCase(userNickName) && !md.Solved) >= TwitchPlaySettings.data.ModuleClaimLimit
+			&& !(!TwitchGame.Instance.Modules.Where(module => module.PlayerName != null && module.PlayerName.EqualsIgnoreCase(userNickName) && !module.Solved)
+			.Any(module => TwitchGame.Instance.CommandQueue.Find(item => item.Message.Text.StartsWith(('!' + module.Code + ' ').ToString())) == null) && TwitchPlaySettings.data.QueuedClaimOverride)
```

Comment:
This would be a bit simplier:
```suggestion
			&& (TwitchGame.Instance.Modules.Any(module => module.PlayerName != null && module.PlayerName.EqualsIgnoreCase(userNickName) && !module.Solved && TwitchGame.Instance.CommandQueue.All(item => !item.Message.Text.StartsWith($"!{module.Code} "))) || !TwitchPlaySettings.data.QueuedClaimOverride)
```